_includes/people/mills.b.html

```diff
@@ -1,15 +1,16 @@
 <div class="media">
   <img class="media-object pull-left" src="{{page.root}}/img/people/mills.b.jpg" alt="Bill Mills" />
   <div id="mills.b" class="media-body">
-    <span class="person">Bill Mills</span> is a physicist and software
-    developer currently working for TRIUMF, Canada's national lab for
-    nuclear and particle physics.  Bill's current focus is on
-    overhauling research methodology both in terms of software
-    development and use, and with regards to UI and UX on
-    next-generation many-user experiments.  He is a co-founder of
-    <a href="http://interdisciplinaryprogramming.com/">InterdisciplinaryProgramming.com</a>,
-    part of his ongoing advocacy of both greater collaboration between
-    scientists and software developers, and of open source software
-    development practices for the scientific community.
+    <span class="person">Bill Mills</span> is a physicist and community manager 
```

Comment:
No link?  (If you want to add one, use `<a class="person" href="whatever">your name</a>`.)